derive_builder_core/src/setter.rs

```diff
@@ -151,7 +154,26 @@ impl<'a> ToTokens for Setter<'a> {
                         let mut new = #self_into_return_ty;
                         new.#field_ident = ::derive_builder::export::core::option::Option::Some(converted);
                         Ok(new)
-                }));
+                    }
+                ));
+            }
+
+            if let Some(ref ident_each) = self.each {
+                tokens.append_all(quote!(
+                    #(#attrs)*
+                    #[allow(unused_mut)]
+                    #vis fn #ident_each <ITEM>(#self_param, item: ITEM) -> #return_ty
+                    where
+                        #ty: ::derive_builder::export::core::default::Default + ::derive_builder::export::core::iter::Extend<ITEM>,
+                    {
+                        #deprecation_notes
+                        let mut new = #self_into_return_ty;
+                        new.#field_ident
+                            .get_or_insert_with(|| ::derive_builder::export::core::default::Default::default())
```

Comment:
```suggestion
                            .get_or_insert_with(::derive_builder::export::core::default::Default::default)
```
This would satisfy the [redundant_closure](https://rust-lang.github.io/rust-clippy/v0.0.212/#redundant_closure) clippy lint, but I've seen cases in the past where that lint would recommend invalid code so you should probably double-check the change doesn't break anything.

Reply:
Done